src/frontend/web_application/src/components/MultiSelectFieldGroup/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { v1 as uuidV1 } from 'uuid';
+import classnames from 'classnames';
+import FieldGroup from '../FieldGroup';
+
+const propTypeOption = PropTypes.oneOfType([PropTypes.string, PropTypes.number]);
+const alphaNumPropType = PropTypes.oneOfType([PropTypes.string, PropTypes.number]);
+
+class MultiSelectFieldGroup extends PureComponent {
```

Comment:
it looks like it is unused

Reply:
I removed references to it, a bit too quickly as it seems.